modules/opr/parse.go

```diff
@@ -7,31 +7,43 @@ import (
 
 // Parse takes any input and attempts to automatically determine the version
 func Parse(data []byte) (OPR, error) {
-	p, err := ParseV2Content(data)
-	if err == nil {
-		return p, nil
-	}
+	// The order v1, then v2 MATTERS.
+	// When doing v2 then v1, there is valid json that decodes successfully into
+	// the protobug opr.
+	// See TestStrangeVector for the example. Since json is only valid for ascii
+	// space, it's more likely to fail given protobuf data.
```

Comment:
that's weird, i definitely assumed that protobuf would always fail on json input, but i can see how the "repeated" keywords might get it to accept a surplus input as valid.